crates/testing/src/helpers.rs

```diff
@@ -145,10 +137,10 @@ pub async fn build_cert<
     V: Versions,
     DATAType: Committable + Clone + Eq + Hash + Serialize + Debug + 'static,
     VOTE: Vote<TYPES, Commitment = DATAType>,
-    CERT: Certificate<TYPES, Voteable = VOTE::Commitment>,
+    CERT: Certificate<TYPES, VOTE::Commitment, Voteable = VOTE::Commitment>,
 >(
     data: DATAType,
-    membership: &TYPES::Membership,
+    da_membership: &TYPES::Membership,
```

Comment:
I think this should take the stake table + threshold directly (and therefore `build_assembled_sig` should too)

this isn't used in the integration tests like `libp2p::libp2p_network` though, so shouldn't have anything to do with the failure

Reply:
rethinking my comment here, will leave some thoughts on zulip